src/main/java/au/gov/geoscience/portal/server/services/filters/PetroleumTenementFilter.java

```diff
@@ -70,15 +69,24 @@ public String getFilterStringAllRecords() {
     }
 
     public String getFilterStringBoundingBox(FilterBoundingBox bbox) {
-        List<String> localFragment = new ArrayList<String>(fragments);
+        List<String> localFragment = new ArrayList<>(fragments);
         if (bbox != null) {
             localFragment.add(this.generateBboxFragment(bbox, "pt:shape"));
         }
         return this.generateFilter(this.generateAndComparisonFragment(localFragment.toArray(new String[localFragment.size()])));
     }
 
     public String getFilterWithAdditionalStyle() {
-        List<String> localFragment = new ArrayList<String>(fragments);
+        List<String> localFragment = new ArrayList<>(fragments);
         return this.generateFilter(this.generateAndComparisonFragment(localFragment.toArray(new String[localFragment.size()])));
     }
+
+    // The Northern Territory services returns owner/holder names with a \n in front that causes the filter to fail
```

Comment:
This comment potentially needs a bit more information, like causes the filter to fail when using AbstractFilter.generatePropertyIsLikeFragment(), or whatever the case may be.

Reply:
Actually probably something like "Filter a column, ignoring newline characters. This is required for the Northern Territory services due to them containing a a \n character."